go-getting-started/controllers/insert_test.go

```diff
@@ -0,0 +1,108 @@
+package controllers
+
+import (
+	"testing"
+
+	"github.com/heroku/go-getting-started/config"
+	"github.com/heroku/go-getting-started/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestControllers_InsertData(t *testing.T) {
+	tt := []struct {
+		name string
+		want models.Joke
+	}{
+		{
+			name: "Big number",
+			want: models.Joke{
+				Body:  "asdada",
+				ID:    "sdfadsa",
+				Score: 4,
+				Title: "asd",
+			},
+		},
+		{
+			name: "less than zero",
+			want: models.Joke{
+				Body:  "asdada",
+				ID:    "sad",
+				Score: 4,
+				Title: "asd",
+			},
+		},
+		{
+			name: "zero",
+			want: models.Joke{
+				Body:  "asdaa",
+				ID:    "gsdfsf",
+				Score: 4,
+				Title: "asd",
+			},
+		},
+		{
+			name: "Empty title",
+			want: models.Joke{
+				Body:  "asdada",
+				ID:    "rdsf",
+				Score: 4,
+				Title: "asd",
+			},
+		},
+	}
+
+	for _, tc := range tt {
+		t.Run(tc.name, func(t *testing.T) {
+			var joke models.Joke
+
+			collection := config.ConnectDB("mongodb://localhost:27017")
```

Comment:
You don't have to connect to database in each iteration of the loop. Connect to it once before entering the loop. Ultimately, it would be good if you create a constant for the URL.